python_modules/libraries/dagster-aws/dagster_aws/ecs/tasks.py

```diff
@@ -80,9 +86,8 @@ def task_definition_dict(self):
                 ),
                 *self.sidecars,
             ],
-            # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
-            cpu="256",
-            memory="512",
+            cpu=self.cpu,
+            memory=self.memory,
```

Comment:
not sure if we want validation for the cpu/mem combos https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html worst case it'll just fail upstream in ecs